src/components/Differential/TabMultiFeature.jsx

```diff
@@ -69,6 +71,16 @@ class TabMultiFeature extends Component {
         if (s) {
           const srcUrl = `${s.svg}${dimensions}`;
           const isPlotlyPlot = s.plotType.plotType.includes('plotly');
+          const isMultiModelMultiTestVar = isMultiModelMultiTest(
+            s.plotType.plotType,
+          );
+          const testIdNotCommon =
+            !differentialTestIdsCommon.includes(differentialTest);
+          const errorMessagePlotlyMultiFeature =
+            isMultiModelMultiTestVar && testIdNotCommon
+              ? `${s.plotType.plotDisplay} can not be created because the currently selected test is not present in all models`
```

Comment:
distinguish error message between when plot isn't available for selected features vs. not available because current test isn't common across all models